.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Cần tạo file .gitignore cho toàn dự án ở phía ngoài.

Reply:
Cho các thư mục và file không cần thiết như .idea, Test1-MonstarLab.iml, target/classes vào .gitignore